Assets/Scripts/Utilities/Logger.cs

```diff
@@ -30,7 +30,7 @@ public static void LogException(Exception exception)
         }
     }
 
-    public static void LogAssertion(string message)
+    public static void LogAssertion(object message)
```

Comment:
I also agree that object should be used, not string. Requiring a string only forces the user of the log to convert something to a string first. Having it as an object will produce the same thing as calling the .ToString() function, which makes sense.